src/components/Modal-custom-btn/ModalCustomFormRadius.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { Dispatch, FC, SetStateAction } from 'react'
+import InfoConfirmationModal from './ModalCustomWindow'
+import { Typography } from '@mui/material'
+import { Button } from '../UI/Button'
+import { SubmitHandler, useForm } from 'react-hook-form'
+import { yupResolver } from '@hookform/resolvers/yup'
+import * as Yup from 'yup'
+import RadioForm from '../RadioForm/RadioForm'
+
+interface ModalCustomFormRadiusProps {
```

Comment:
you do not need `interface` here, use `type`

Reply:
done